Scripts/CI/README_Metadata_StyleCheck/entry.py

```diff
@@ -0,0 +1,564 @@
+import os
+import argparse
+import subprocess as sp
+from os import listdir
+from os.path import isfile, join
+import json
+
+def main():
+    # Tell the program that we're using global variables
+    msg = 'Entry point of the docker to run json style check scripts.'
+    parser = argparse.ArgumentParser(description=msg)
+    parser.add_argument('-s', '--string', help='A JSON array of file paths.')
+    args = parser.parse_args()
+    cleanstring = ""
+    try:
+        for letter in args.string:
+            if letter not in '[" \']':
+                cleanstring+=letter
+        file_list = cleanstring.split(",")
+    except Exception as e:
+        print(f"Unable to split args {args.string}.\nException: {e}")
+        exit(1)
+    check_files(file_list)
+    
+def check_files(file_list):
+    print("Files to check: ")
+    for file in file_list:
+        print(file)
+    print("\n")
+    total_errors = 0
+    if not file_list:
+        print("No files in file set")
```

Comment:
consider an error message that might be more descriptive to a user of the script so they know what to change. for example "the provided file paths contain no markdown or json files"